app/controllers/dish_ingredients_controller.rb

```diff
@@ -0,0 +1,19 @@
+class DishIngredientsController < ApplicationController
+  def create
+    dish = Dish.find(params[:dishes_id])
+    ingredient = Ingredient.find(params[:ingredient_id])
+    DishIngredient.create!(dish: dish, ingredient: ingredient)
+  
+    redirect_to "/dishes/#{dish.id}"
+  end
+
+  def destroy
```

Comment:
Nice work getting through this extension! Great RESTful implementation of both of these CRUD actions on a joins resource